### PR TITLE
[ADD] helpdesk: doc about reports

### PR DESCRIPTION
**Pull request by kbr-odoo**
Added doc about reports.

### CODE REVIEW

File: helpdesk/overview/reports.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+.. image:: media/report_time_to_close.png
+   :align: center
+   :alt: View of helpdesk ticket analysis of the hours to close by ticket type and team in
+         Odoo Helpdesk
+
+Save filters
+~~~~~~~~~~~~
+
+Save the filters you use the most and save time not having to construct them every time they are
+needed. To do so, set the groups, filters and measures needed, then go to *Favorites*.
```
Comment (Lau-Odoo) [inline helpdesk/overview/reports.rst:52]:
```suggestion
Save the filters you use the most and avoid having to reconstruct them every time they are
needed. To do so, set the groups, filters, and measures needed. Then, go to *Favorites*.
```

**Comment by kbr-odoo** [comment]
@Lau-Odoo Thanks!

File: helpdesk/overview/reports.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
+============================
+Reports for a Better Support
+============================
+
+An efficient customer service solution also has a built-in reporting option. Reports allow you to
```
Comment (jubodoo) [inline helpdesk/overview/reports.rst:5]: Should have, not also has.

File: helpdesk/overview/reports.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+track trends, identify areas for improvement, manage employees’ workloads and, most importantly,
+meet your customer’s expectations.
+
+Cases
+~~~~~
+
+Some examples of the possible reports Odoo Helpdesk can generate include:
```
Comment (jubodoo) [inline helpdesk/overview/reports.rst:12]:
```suggestion
Some examples of the reports Odoo Helpdesk can generate include:
```

**Comment by kbr-odoo** [comment]
@jubodoo thanks! To be merged.

**Pull request merged**